server/cli/cli.py

```diff
@@ -29,3 +30,4 @@ def cli(upgrade_check):
 
 cli.add_command(launch)
 cli.add_command(prepare)
+cli.add_command(config)
```

Comment:
Why add this?  Unless I am misunderstanding, all it can do is display the internal default config.  Several confusions about this from a UI design standpoint:
- the config is only relevant to the launch command, which is not being run (eg, it doesn't configure `prepare`)
- the names/default values are not the same you see when you run `cellxgene launch --help`

Is the idea that it would give the user a hint about what the config file should look like, if they want to use `cellxgene launch -c <file>`?

If so, I suggest:
- it should be an option on the `launch` command, as that is what it is showing the config for
- it should have a name like `cellxgene launch --dump-default-config` (print it and exit)

Having a separate top-level sub-command doesn't really make sense given the current command structure.

Reply:
Okay, I'll make it part of launch.   Yes, the idea the user will use it like this:

> cellxgene launch --dump_default_config > myconfig.yaml
> vi myconfig.yaml  # customize for my environment 
> cellxgene launch -c myconfig.yaml